src/cascadia/TerminalApp/App.cpp

```diff
@@ -25,6 +25,67 @@ namespace winrt
     using IInspectable = Windows::Foundation::IInspectable;
 }
 
+// clang-format off
+static const std::unordered_map<::TerminalApp::SettingsLoadWarnings, std::wstring_view> settingsLoadWarningsLabels {

```

Comment:
more global runtime initializers!

Reply:
we could make them constexpr `std::arrays` and then just linear search them. /shrug

Reply:
theoretically if they were kept in order with the enum, we could index into the array with the enum value itself 🤔

Reply:
_hmm_ this is true and perhaps even great